package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "mentorship-page",
```

Comment:
@IvanFengJK Revert any changes made to this file before pushing to github. `package-lock.json` should be left untouched.